src/tests/ftest/util/ior_utils.py

```diff
@@ -238,3 +239,63 @@ def get_default_env(self, manager_cmd, attach_info, log_file=None):
             env["IOR_HINT__MPI__romio_daos_obj_class"] = self.daos_oclass.value
 
         return env
+
+    @staticmethod
+    def get_ior_metrics(cmdresult):
+        """Parse the CmdResult (output of the test) and look for
+           the ior stdout and get the read and write metrics.
+
+        Args:
+            cmdresult (CmdResult): output of job manager
+
+       Returns:
+            metrics (tuple) : list of write and read metrics from ior run
+
+        """
+        ior_metric_summary = "Summary of all tests:"
+        messages = cmdresult.stdout.splitlines()
+        # Get the index whre the summary starts and add one to
+        # get to the header.
+        idx = messages.index(ior_metric_summary)
+        # idx + 1 is header.
+        # idx +2 and idx + 3 will give the write and read metrics.
+        write_metrics = (" ".join(messages[idx+2].split())).split()
+        read_metrics = (" ".join(messages[idx+3].split())).split()
+
+        return (write_metrics, read_metrics)
+
+class IorMetrics(IntEnum):
+    """
+    Index Name and Number of each column in IOR result summary.
```

Comment:
Isn't this line a docstring?  I'm not sure why pylint is complaining here?  Maybe it needs to be a # comment?

Reply:
I'm not sure where you're seeing this pylint issue but I guess the text "Index Name .." should start right after the quote and not the next line.